features/backdrop.yml

```diff
@@ -0,0 +1,6 @@
+name: "::backdrop"
+description: The `::backdrop` CSS pseudo-element is a box underneath an element, such as a `<dialog>`, that displays in the top layer. It can be used to create obscuring effects distinguishing a modal element from the elements underneath.
```

Comment:
```suggestion
description: The `::backdrop` CSS pseudo-element is a box underneath an element in the top layer, such as a `<dialog>`. It can be used to create obscuring effects distinguishing a modal element from the elements underneath.
```